src/python/pants/backend/python/dependency_inference/scripts/import_parser.py

```diff
@@ -31,12 +31,15 @@ class AstVisitor(ast.NodeVisitor):
     def __init__(self, package_parts, contents):
         self._package_parts = package_parts
         self._contents_lines = contents.decode(errors="ignore").splitlines()
+
+        # Each of these maps module_name to first lineno of occurance
         # N.B. use `setdefault` when adding imports
-        self.imports = {}  # maps module_name to first lineno of occurance
+        self.definite_imports = {}
```

Comment:
Probably `strong_imports` if you go with `weak_imports`. Otherwise, definite works.